src/components/configuration/ConfigBar.tsx

```diff
@@ -54,37 +58,20 @@ function ConfigBar({
 
     return (
         <>
-            {isLoading ? (
```

Comment:
Why has the progress indicator for `isLoading` been removed?

Reply:
I felt like the fact that this loading is always super speedy warrants removing it. It just adds layout shift IMO. But not super strongly opinionated about this

Reply:
Hmm, that is a fair point :+1: